lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -365,7 +369,7 @@ newStakePoolLayer tr block0H getEpCst db@DBLayer{..} nl metadataDir = StakePoolL
                         Ranking.desirability epConstants $ Ranking.Pool
                             (unsafeMkRatio 0) -- pool leader pledge
                             poolCost
-                            (unsafeMkRatio $ fromIntegral (getPercentage poolMargin) / 100)
+                            (unsafeMkRatio . fromRational . toRational $ poolMargin)
```

Comment:
What type has `poolMargin` here? It seems to already be a ratio, that we convert 3 times and back into a ratio :thinking: ?

Reply:
Pool margin is a `Ratio Word64`.

unsafeMkRatio is a `Double -> Cardano.Pool.Ranking.Ratio` 😬

Reply:
Note that this is for the desirability calculation, and not to display the pool margin.

Reply:
Why having a `Ratio` type here that has exactly the same purpose as the `Percentage` type :thinking: ?